src/features/common/Accordion.tsx

```diff
@@ -0,0 +1,53 @@
+import * as Accordion from "@radix-ui/react-accordion";
+import { MdChevronRight } from "react-icons/md";
+import { twJoin } from "tailwind-merge";
+import { ReactNode, useState } from "react";
+
+interface AccordionComponentProps {
+  rootStyles: string;
+  type?: "single" | "multiple";
+  itemId: string;
+  trigger: ReactNode;
+  content: ReactNode;
+}
+
+const AccordionComponent = ({
```

Comment:
I personally prefer `export const AccordionComponent` and then using `import { AccordionComponent } from '../whatever/path/'` - this is just preference but as most of the other components use this pattern I think we should do it.

I also don't really like that it's called `AccordionComponent` - I would rename it to just `Accordion`, and then either `import { Root, Item, etc } from "@radix-ui/react-accordion"` or `import * as RadixAccordion`. If you do the latter don't forget you can use F2 to rename a variable in typescript.

Reply:
I'll use `export const Component` syntax going forward